src/clients/mocks/MockEvents.ts

```diff
@@ -0,0 +1,76 @@
+import { Result } from "@ethersproject/abi";
+import { utils as ethersUtils, Event, providers } from "ethers";
+import { random } from "lodash";
+
+const { id, keccak256, toUtf8Bytes } = ethersUtils;
+
+type Block = providers.Block;
+type TransactionResponse = providers.TransactionResponse;
+type TransactionReceipt = providers.TransactionReceipt;
+
+type EthersEventTemplate = {
+  address: string;
+  event: string;
+  topics: string[];
+  args: Result;
```

Comment:
Additional change: Swap the type annotation of args for the actual ethers `Result` type (weird readonly hybrid Object/Array thing).

At this stage I haven't seen any errors or warnings from doing this, so I think we should persist with this definition unless/until something breaks.